ntcore/src/generate/types.json

```diff
@@ -0,0 +1,366 @@
+[
+    {
+        "TypeName": "Boolean",
+        "TypeString": "\"boolean\"",
+        "c": {
+            "ValueType": "NT_Bool",
+            "ParamType": "NT_Bool"
+        },
+        "cpp": {
+            "ValueType": "bool",
+            "ParamType": "bool",
+            "TYPE_NAME": "BOOLEAN"
+        },
+        "java": {
+            "ValueType": "boolean",
+            "EmptyValue": "false",
+            "ConsumerFunctionPackage": "edu.wpi.first.util.function",
+            "FunctionTypePrefix": "Boolean",
+            "ToWrapObject": "Boolean.valueOf",
+            "FromStorageBegin": "(Boolean) "
+        },
+        "jni": {
+            "jtype": "jboolean",
+            "jtypestr": "Z",
+            "JavaObject": false,
+            "FromJavaBegin": "",
+            "FromJavaEnd": " != JNI_FALSE",
+            "ToJavaBegin": "static_cast<jboolean>(",
+            "ToJavaEnd": ")",
+            "ToJavaArray": "MakeJBooleanArray"
+        }
+    },
+    {
+        "TypeName": "Integer",
+        "TypeString": "\"int\"",
+        "c": {
+            "ValueType": "int64_t",
+            "ParamType": "int64_t"
+        },
+        "cpp": {
+            "ValueType": "int64_t",
+            "ParamType": "int64_t",
+            "TYPE_NAME": "INTEGER"
+        },
+        "java": {
+            "ValueType": "long",
+            "EmptyValue": "0",
+            "FunctionTypePrefix": "Long",
+            "ToWrapObject": "Long.valueOf",
+            "FromStorageBegin": "((Number) ",
+            "FromStorageEnd": ").longValue()"
+        },
+        "jni": {
+            "jtype": "jlong",
+            "jtypestr": "J",
+            "JavaObject": false,
+            "FromJavaBegin": "",
+            "FromJavaEnd": "",
+            "ToJavaBegin": "static_cast<jlong>(",
+            "ToJavaEnd": ")",
+            "ToJavaArray": "MakeJLongArray"
+        }
+    },
+    {
+        "TypeName": "Float",
+        "TypeString": "\"float\"",
```

Comment:
If the escaped quotes are in all types, why not put them in the template?

Reply:
Because there's places in the templates it's getting replaced with a variable/parameter name instead of being a string.